.github/actions/install-dashboards/action.yml

```diff
@@ -53,7 +50,7 @@ runs:
       shell: bash
```

Comment:
Like above, I know you didn't make these changes but do you think that these number should be incremented now that we are looking at 2.5. and 2.4 intsead?